.github/workflows/ci.yml

```diff
@@ -587,6 +587,10 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           merge-multiple: true
+      # Having multiple artifacts named the same might be onfusing to the users. Drop the unsigned versions
```

Comment:
Typo on "onfusing"

Reply:
Not yet addressed.

```suggestion
      # Having multiple artifacts named the same might be confusing to the users. Drop the unsigned versions.
```

Reply:
Odd, I did fix it in the web version of vscode (https://GitHub.dev), but it must've been lost somehow when commiting.